source/2019-11-22-the-ember-times-issue-125.md

```diff
@@ -96,13 +95,11 @@ Both beta versions require [Ember 3.13](https://blog.emberjs.com/2019/09/25/embe
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Ember Weekend Episode 133: Bring Your Own Babel 🎧](https://emberweekend.com/episodes/bring-your-own-babel/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Following a *hiatus* 😉, [Jonathan Jackson](https://twitter.com/rondale_sc), [Robert Jackson](https://twitter.com/rwjblue), and [Chase McCarthy](https://twitter.com/code0100fun) got together to record a [new Ember Weekend episode](https://emberweekend.com/episodes/bring-your-own-babel/) and share news in the Ember ecosystem.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In episode 133, you will learn more about the remaining tasks for [Octane](https://emberjs.com/editions/octane) release, improvements made in [`ember-angle-brackets-codemod`](https://github.com/ember-codemods/ember-angle-brackets-codemod) and [ember-no-implicit-this-codemod](https://github.com/ember-codemods/ember-no-implicit-this-codemod), and how you can join and help grow the [ember-codemods organization](https://github.com/ember-codemods)!
```

Comment:
I think here it'd be

```suggestion
In episode 133, you will learn more about the remaining tasks for the [Octane](https://emberjs.com/editions/octane) release, improvements made in [`ember-angle-brackets-codemod`](https://github.com/ember-codemods/ember-angle-brackets-codemod) and [ember-no-implicit-this-codemod](https://github.com/ember-codemods/ember-no-implicit-this-codemod), and how you can join and help grow the [ember-codemods organization](https://github.com/ember-codemods)!
```

What do you think?

Reply:
Yep, I think it helps with a reader's understanding of the sentence. Thanks!